### PR TITLE
Configure Rust Core Lib with specified logging level

### PR DESCRIPTION
**Pull request by duskoo**
```
CoreLogLevel: 0 -> Trace, ... 4 -> Error
let loggerSetupStatus = setup_logger(CoreLogLevel.init(0))
```

### CODE REVIEW

File: CoEpi/native/NativeCore.swift [hunk 1]
```diff
@@ -41,7 +41,10 @@ protocol TcnGenerator {
 extension NativeCore: ServicesBootstrapper {
 
     // WARN: Must not run long operations. Currently executed synchronously during startup.
-    func bootstrap(dbPath: String) -> Result<(), ServicesError> {        
+    func bootstrap(dbPath: String) -> Result<(), ServicesError> {
+        //CoreLogLevel: 0 -> Trace, ... 4 -> Error; coepi_only logs -> true/false
+        let loggerSetupStatus = setup_logger(CoreLogLevel.init(0), true)
```
Comment (astewart43-gtri) [inline CoEpi/native/NativeCore.swift]: When I try to run, I get an error on "setup_logger"
am I missing a setup somewhere?

Comment (duskoo) [reply CoEpi/native/NativeCore.swift]: @astewart43-gtri Corresponding Rust branch has not been merged/released. To properly run this PR, you need to checkout https://github.com/Co-Epi/app-backend-rust/tree/feature/logging-parametrized and build the Rust lib manually following the instructions on the wiki: https://github.com/Co-Epi/app-backend-rust/wiki/Building-library-for-iOS

**Review by astewart43-gtri — APPROVED** [review]

File: CoEpi/native/NativeCore.swift (hunk 1 above)
Comment (astewart43-gtri) [reply CoEpi/native/NativeCore.swift]: does this need to wait till the rust branch is merged?

File: FFIStandaloneTests/FFIStandaloneTests.swift [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        for i:Int32 in 0...4 {
+            loggerSetupWithLevel(levelInt: i)
+        }
+    }
+    
+    func loggerSetupWithLevel(levelInt: Int32) {
+        let level = CoreLogLevel(levelInt)
+        print("iOS : Logger level : \(level)")
+        let ret = setup_logger(level, false);
+        XCTAssertEqual(levelInt, ret);
```
Comment (ivnsch) [inline FFIStandaloneTests/FFIStandaloneTests.swift]: This is a bit weird. The calls after the first have no effect, but in Rust you just return the level you passed as a parameter.

**Review by ivnsch — CHANGES_REQUESTED** [review]
I'm actually requesting changes because it's confusing. Either return the actual log level in Rust or don't return the log level.

File: FFIStandaloneTests/FFIStandaloneTests.swift (hunk 2 above)
Comment (duskoo) [reply FFIStandaloneTests/FFIStandaloneTests.swift]: Ive updated setup_logger to return max_level

**Review by ivnsch — APPROVED** [review]

**Pull request merged**